sale_order_revision/__init__.py

```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from . import models
+
+
+def populate_unrevisioned_name(cr, registry):
```

Comment:
This should be moved into `hooks.py` and the function renamed to `post_init_hook`. You can then add `from.hooks import post_init_hook` here. cf what's done in https://github.com/OCA/maintainer-tools/tree/master/template/module